Jellyfin.Server/Properties/launchSettings.json

```diff
@@ -6,7 +6,8 @@
       "applicationUrl": "http://localhost:8096",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }
+      },
+      "commandLineArgs": "--webdir \"C:\\Users\\Cody\\Code\\Jellyfin\\jellyfin-web\\dist\""
```

Comment:
```suggestion
      }
```

Reply:
Wonder how many more times I'll do that... 🙃